src/parsers/babel_parser.js

```diff
@@ -171,3 +170,34 @@ export const parse = (file: string, data: ?string): ParseResult => {
 
   return parseResult;
 };
+
+const additionalPlugins = [
+  'jsx',
+  'asyncGenerators',
+  'bigInt',
+  'classProperties',
+  'classPrivateProperties',
+  'classPrivateMethods',
+  'doExpressions',
+  'dynamicImport',
+  'exportDefaultFrom',
+  'exportNamespaceFrom',
+  'functionBind',
+  'functionSent',
+  'importMeta',
+  'logicalAssignment',
+  'nullishCoalescingOperator',
+  'numericSeparator',
+  'objectRestSpread',
+  'optionalCatchBinding',
+  'optionalChaining',
+  'partialApplication',
+  'throwExpressions',
+  'topLevelAwait',
+];
+
+// Its not possible to use the parser with flow and typescript active at the same time
+export const parseJs = (file: string, data: ?string): ParseResult =>
+  parse(file, data, {plugins: [...additionalPlugins, 'flow']});
+export const parseTs = (file: string, data: ?string): ParseResult =>
+  parse(file, data, {plugins: [...additionalPlugins, 'typescript']});
```

Comment:
wildcard plugin list no longer supported in babel-7. The current decision is to include them *all*, or as much as possible. Yes as new plugins become available and our users start to use them, we might need to come back here to add them or consider a way for users to pass their list of plugins. 

We could deal with that later... if it happened as infrequent as it was then manually maintaining a full list might not be too bad. But I think **we should call out the plugins we currently support in the README file.**

Reply:
we could add addtionalPlugins as param to parse? then it would be more dynamic for the API user. also we would not have to adjust the package if a new proposal was released.